frontend/src/pages/HomePage.jsx

```diff
@@ -79,30 +50,11 @@ class HomePage extends Component {
             <Typography
               type="title"
               color="inherit"
-              style={{
-                flex: 1
-              }}
-            />
+              style={{ flex: 1 }
+              } />
```

Comment:
Is this normal? lol

Reply:
Just noticed I messed with the brackets. Oops. No, I don't think it's normal to style curly braces like this. I'll make another change.